shared/tests/utility_module/actor_test.go

```diff
@@ -24,14 +24,14 @@ func TestUtilityContext_HandleMessageStakeApp(t *testing.T) {
 	if err := ctx.SetAccountAmount(out, defaultAmount); err != nil {
```

Comment:
All the tests in `actor_test.go` are still named `TestUtilityContext_*App`.

Is it intentional? If not, please update